docs/Typography.stories.tsx

```diff
@@ -1,4 +1,5 @@
-import { ComponentStory, ComponentMeta } from '@storybook/react';
```

Comment:
Just updating deprecated components in Typography story will save a docs update for another PR to keep this one a reasonable size